src/components/Filters.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import { Box, Button, Checkbox, FormControlLabel, Grid, Switch, Typography } from '@mui/material';
+import Tags from './Tags';
+
+interface FiltersProps {
+  allTags: string[];
+  selectedTags: string[];
+  handleTagClick: (tag: string) => void;
+  selectedLicenses: string[];
+  handleLicenseChange: (license: string) => void;
+  selectedSources: string[];
+  handleSourcesChange: (source: string) => void;
+  maintainedByOasis: boolean;
+  handleMaintainedByOasisToggle: () => void;
+  handleClearTags: () => void;
+}
+
+const Filters: React.FC<FiltersProps> = ({
```

Comment:
Could we hide the Filters component by default? It takes roughly half the example card height of space. I would show a filter toggle button next to the search field.